### ISSUE TITLE
Incompatible Packages

### ISSUE THREAD

**Issue by AbhijeetShk** [anikethsaha/MERN-Boilerplate#169]
The versions of @babel/core and @babel/preset-es2015 in package.json are incompatible causing the application to break and thus npm install is not working.

**TODO-**

1. Update both packages to the latest version.
2. Remove the package unless it doesn't break the application.

**Comment by Vidhanvyrs** [comment]
hey @Programmer-Dost can i work on this issue

**Comment by AbhijeetShk** [comment]
Yeah Sure. Although no one is maintaining this repo for a long time.
